src/assets/css/fonts.scss

```diff
@@ -1,5 +1,7 @@
+@import "../../styles/noto-sans/index.scss";
+
 @mixin font($weight: null, $size: null) {
-    font-family: "Noto Sans", sans-serif;
+    font-family: "Noto Sans", "Noto Sans HK", "Noto Sans JP", "Noto Sans KR", "Noto Sans SC", sans-serif;
```

Comment:
Warning that some of these have conflicting characters (JP and SC, from memory).

Some of the kanji are different between the two scripts, but share the same Unicode number.

Reply:
Yeah I found this issue https://github.com/jellyfin/jellyfin-web/issues/913 and read through the notes there. I guess we should add a setting to set a hierarchy or preferred character set somewhere.

Reply:
KR also has all the Hanja (full characters) in it, so this needs to be set on a per language basis.

Reply:
I was thinking we probably shouldn't just rely on the display language since some people may have a larger amount of content in a specific language that is different than their display language. Then they would probably have a preference for which language should be preferred.

This _could_ be done with custom css until a proper preference is added.